docs/rules/no-restricted-syntax.md

```diff
@@ -1,16 +1,16 @@
-# Disallow certain syntax (no-restricted-syntax)
+# disallow specified syntax (no-restricted-syntax)
 
 JavaScript has a lot of language features, and not everyone likes every features. As a result, some projects choose to disallow the use of certain language features altogether. For instance, you might decide to disallow the use of `try-catch` or `class`.
 
 Rather than creating separate rules for every language feature you want to turn off, this rule allows you to configure the syntax elements you want to restrict use of. These elements are represented by their [ESTree](https://github.com/estree/estree) node types. For example, a function declaration is represented by `FunctionDeclaration` and the `with` statement is represented by `WithStatement`. You may find the full list of AST node names you can use [on GitHub](https://github.com/eslint/espree/blob/master/lib/ast-node-types.js) and use the [online parser](http://eslint.org/parser/) to see what type of nodes your code consists of.
 
 ## Rule Details
 
-This rule is aimed at eliminating certain syntax from your JavaScript. As such, it warns whenever it sees a node type that is restricted by its options.
+This rule disallows specified (i.e. user-defined) syntax.
 
 ## Options
 
-This rule takes a list of strings where strings denote the node types:
+This rule takes a list of strings:
```

Comment:
That's quite a bit different, though. It is a single option (with multiple known properties). So, I don't think it helps much for this sentence, and it's a lot of work (and maintenance) to list each type that can be used. I don't remember off the top of my head, but do we have any other options that are more closely used similar to this that are already done?